code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -223,5 +196,128 @@
 			//Prevents whatever mechanism is causing it to hit negative temperatures.
 		return REACTING
 
+/datum/gas_reaction/brownsformation //The formation of brown gas. Endothermic.
+	priority = 3
+	name = "Brown Gas formation"
+	id = "brownsformation"
+
+/datum/gas_reaction/brownsformation/init_reqs()
+	min_requirements = list(
+		"oxygen" = 20,
+		"nitrogen" = 20,
+		"temp" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST*200
+	)
+
+/datum/gas_reaction/brownsformation/react(datum/gas_mixture/air)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = temperature/FIRE_MINIMUM_TEMPERATURE_TO_EXIST*100
+	var/energy_used = heat_efficency*BROWNS_FORMATION_ENERGY
+	air.assert_gases("oxygen","nitrogen","browns")
+
+	cached_gases["oxygen"][MOLES] -= heat_efficency
+	cached_gases["nitrogen"][MOLES] -= heat_efficency
+	cached_gases["browns"][MOLES] += heat_efficency*2
+
+	if(energy_used > 0)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature*old_heat_capacity - energy_used)/new_heat_capacity),TCMB)
+		return REACTING
+
+/datum/gas_reaction/bzformation //Formation of BZ by combining plasma and tritium at low pressures. Exothermic.
+	priority = 3
+	name = "BZ Gas formation"
+	id = "bzformation"
+
+/datum/gas_reaction/bzformation/init_reqs()
+	min_requirements = list(
+		"tritium" = 10,
+		"plasma" = 10
+	)
+
+
+/datum/gas_reaction/bzformation/react(datum/gas_mixture/air)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/pressure = air.return_pressure()
+
+	var/old_heat_capacity = air.heat_capacity()
+	var/reaction_efficency = pressure/0.1*ONE_ATMOSPHERE
+	var/energy_released = 2*reaction_efficency*FIRE_CARBON_ENERGY_RELEASED
+
+
+	air.assert_gases("tritium","plasma","BZ")
+	cached_gases["bz"][MOLES]+= reaction_efficency
+	cached_gases["tritium"][MOLES]-= 2*reaction_efficency
+	cached_gases["plasma"][MOLES]-= reaction_efficency
+
+
+	if(energy_released > 0)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature*old_heat_capacity + energy_released)/new_heat_capacity),TCMB)
+		return REACTING
+
+/datum/gas_reaction/stimformation
+	priority = 3
+	name = "Stimulum formation"
+	id = "stimformation"
+/datum/gas_reaction/stimformation/init_reqs()
+	min_requirements = list(
+		"tritium" = 30,
+		"plasma" = 10,
+		"bz" = 20,
+		"browns" = 30,
+		"temp" = STIMULUM_HEAT_SCALE/2)
+
+/datum/gas_reaction/stimformation/react(datum/gas_mixture/air)
+	var/list/cached_gases = air.gases
+
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_scale = air.temperature/STIMULUM_HEAT_SCALE
+	var/stim_energy_change
+	stim_energy_change = (STIMULUM_FIRST_RISE(heat_scale**2)) - (STIMULUM_FIRST_DROP(heat_scale**3)) + (STIMULUM_SECOND_RISE(heat_scale**4)) - (STIMULUM_ABSOLUTE_DROP(heat_scale**5))
```

Comment:
ohh baby we've got polynomials

Reply:
>>>polynomials

S T O P

Reply:
what the fuck

Reply:
I think it'll be fun but I'd love to see a graph

Reply:
https://www.wolframalpha.com/input/?i=0.75x%5E2-0.25x%5E3%2B0.001x%5E4-0.0000062x%5E5+from+0.5+to+5

Reply:
Oh fucked up the constants on that one. I'll update it with the proper equation soon.